kola/cluster/cluster.go

```diff
@@ -29,7 +29,8 @@ import (
 type TestCluster struct {
 	*harness.H
 	platform.Cluster
-	NativeFuncs []string
+	NativeFuncs   []string
+	TorcxManifest *TorcxManifest
```

Comment:
My specific thought is that longer term these actually might not be per-kola-run but per-cluster.

If, for offline tests, kola mirror torcx packages into an http server and provides an endpoint to each testcluster, it's possible (depending on implementation) that it could differ per testcluster.